packages/at_test_proxy/lib/src/message.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:io';
+
+enum MessageStatus {
+  response, // from client to server
+  request, // from server to client
+  none,
+}
```

Comment:
Some of the naming is unnatural because it's a proxy...

e.g. the atServer response comes from the clientSocket to the serverSocket.